baseplate/lib/prometheus_metrics.py

```diff
@@ -438,3 +438,59 @@ def get_requests_total_metric(cls) -> Counter:
     @classmethod
     def get_active_requests_metric(cls) -> Gauge:
         return cls.active_requests
+
+
+class PrometheusSQLAlchemyClientMetrics:
+    prefix = "database_client"
+
+    labels = [
+        "statement",
+    ]
+
+    # Latency histogram of HTTP calls made by clients
+    # buckets are defined above (from 100µs to ~14.9s)
+    latency_seconds = Histogram(
+        f"{prefix}_latency_seconds",
```

Comment:
There is still an outstanding question of whether or not we should have metrics for `trace_*` vs `{prefix}_*`.

Reply:
I like this idea!